BaseStation/BaseServer/BaseServer.cs

```diff
@@ -15,20 +15,23 @@ namespace HuskyRobotics.BaseStation.Server
     /// </summary>
     public static class BaseServer
     {
-
-        private static bool shutdown = false;
-        private static Controller gamepad;
         private static int LeftThumbDeadzone = 7849;
         private static int RightThumbDeadzone = 8689;
         private static int TriggerThreshold = 30;
 
-        public static void Start(Controller controller)
+        private const long CONTROL_SEND_INTERVAL = 100_000_000;
+
+        public static void Setup()
         {
             Scarlet.Communications.Server.Start(1025, 1026);
             Scarlet.Communications.Server.ClientConnectionChange += ClientConnected;
-            Parse.SetParseHandler(0xC0, gpsHandler);
-            Parse.SetParseHandler(0xC1, magnetomerHandler);
-            gamepad = controller;
+            Parse.SetParseHandler(0xC0, GpsHandler);
+            Parse.SetParseHandler(0xC1, MagnetomerHandler);
+        }
+
+        public static void Shutdown()
```

Comment:
Need to explicitly tell the motors to stop when the UI is shutdown. Also if possible to do this explicitly when the UI is crashing and not shutting down cleanly.